backend/Recodo.BLL/Services/FileService.cs

```diff
@@ -35,7 +35,8 @@ public async Task<int> SaveVideo(string token)
                 IsSaving = false, //Rename to IsSaved
                 AuthorId = Convert.ToInt32(authorId),
                 CreatedAt = DateTime.Now,
-                Name = $"Video_{DateTime.Now.Day}{DateTime.Now.Month}{DateTime.Now.Year}_{DateTime.Now.Hour}{DateTime.Now.Minute}",
+                Name = $"Video_{FormatVideoTime(DateTime.Now.Day)}{FormatVideoTime(DateTime.Now.Month)}{DateTime.Now.Year}_" +
+                $"{FormatVideoTime(DateTime.Now.Hour)}{FormatVideoTime(DateTime.Now.Minute)}{FormatVideoTime(DateTime.Now.Second)}",
```

Comment:
It's so messy
Can you just cast Now to string using specified formatter?
https://docs.microsoft.com/en-us/dotnet/standard/base-types/standard-date-and-time-format-strings